CONTRIBUTING.md

```diff
@@ -73,7 +74,7 @@ To change external modules (to add a new module or bump the version), update the
 
 Some tests require a Windows VMs or Linux containers. For Linux tests (recommended, if you're not sure) run `docker-compose up -d --build` from `spec/` directory. For windows tests, execute `vagrant up` from the root of the bolt repo to bring these up with the provided Vagrantfile. Any tests that require this are tagged with `:winrm` or `:ssh` in rspec.
```

Comment:
I think we can change this to `docker-compose -f spec/docker-compose.yml up -d --build` like we have in developer docs and remove the bit about `spec/`. It seems easy to miss.